__tests__/src/components/TicketsTable.test.tsx

```diff
@@ -53,25 +44,9 @@ describe('TicketTable Headers', () => {
 jest.mock('next/link', () => ({ children }) => children);
 
 describe('TicketsTable', () => {
-  const items: ServiceRequestSummary[] = [
-    {
-      id: '1',
-      team_member: 'John Doe',
-      description: 'Cleaning service',
-      created_at: '2023-05-01T12:00:00Z',
-      pet: 'Sparky',
-      client: 'Alice Smith',
-    },
-    {
-      id: '2',
-      team_member: 'Sarah Lee',
-      description: 'Grooming service',
-      created_at: '2023-04-15T10:30:00Z',
-      pet: 'Buddy',
-      client: 'Bob Johnson',
-    },
-  ];
-
+  const items = mockServiceRequestSummaries;
+  const row = items[0];
+  const row2 = items[0];
```

Comment:
I think you meant row2 = items[1], also you could write as
```typescript
const [row, row2] = mockServiceRequestSummaries;
```

Reply:
wow I dont even remember the tests i wrote, thanks I will need more help writing tests